### PR TITLE
New Counter Strike Cheat Sheet

### PR DESCRIPTION
**Pull request by yasharmaster**
https://duck.co/ia/view/counter_strike_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@yasharmaster Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "metadata": {
+        "sourceName": "Counter Strike Wiki",
+        "sourceUrl" : "http://counterstrike.wikia.com/wiki/Controls"
+    },
+
+    "aliases": [
+        "counter strike global offensive",
+        "counter strike 1.6",
+        "counter strike source",
+        "cs",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/counter-strike.json]: I think we should probably remove the "cs" alias, because it could mean many other things.

Comment (yasharmaster) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Yeah right! I will remove that. Is there anything else I can change?

**Comment by tagawa** [comment]
Thank you for this @yasharmaster. There seems to be a problem deploying it to the beta server so I'm checking it now.

**Comment by gaulrobe** [comment]
This looks like it has the potential to conflict with #3239. Should they be combined? Or perhaps renamed/retriggered?

File: share/goodie/cheat_sheets/json/counter-strike.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        "sourceName": "Counter Strike Wiki",
+        "sourceUrl" : "http://counterstrike.wikia.com/wiki/Controls"
+    },
+
+    "aliases": [
+        "counter strike global offensive",
+        "counter strike 1.6",
+        "counter strike source",
+        "cs go",
+        "csgo",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Aha, the "cs go" and "csgo" aliases are conflicting with an existing cheat sheet ( https://beta.duckduckgo.com/?q=csgo+cheat+sheet&ia=cheatsheet ).
Please could you remove them?

Comment (yasharmaster) [inline share/goodie/cheat_sheets/json/counter-strike.json]: Yeah sure.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=counter strike cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/counter_strike_cheat_sheet_mobile.png?EDuAr25Xoi)

**Comment by tagawa** [comment]
OK, this is now ready on the beta server for testing: https://beta.duckduckgo.com/?q=counter+strike+cheat+sheet&ia=cheatsheet&iax=1

**Comment by yasharmaster** [comment]
https://duck.co/ia/view/age_of_empires_3_cheat_sheet

**Comment by gaulrobe** [comment]
@yasharmaster Please make a separate PR for the Age of Empires cheat sheet so that these two can be reviewed independently. Thanks!

**Comment by yasharmaster** [comment]
@gaulrobe the pull requests got merged automatically. I don't know how to separate them. How can I make a new PR for Age of Empires?

**Comment by gaulrobe** [comment]
@yasharmaster I see that you are working on `master` in your fork. It may be worthwhile to make new branches `git checkout -b my_branch_name` for each IA you wish to create. I would suggest creating a new branch that only contains the Age of Empires JSON. You can then remove it from this branch.

**Comment by yasharmaster** [comment]
@gaulrobe Thank you. I will do that.

**Comment by daxtheduck** [comment]
@yasharmaster this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**